ntoskrnl/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ if(MSVC)
     add_target_link_flags(ntoskrnl "/SECTION:.rsrc,!DP") #Accessed from bugcheck code
     add_target_link_flags(ntoskrnl "/SECTION:INIT,ERWD")
 else()
-    if(GDB)
+    if(GDB AND NOT CLANG)
```

Comment:
Why? What's the consequence of not optimizing? Do we get size issues like in freeldr or something?

Reply:
PSEH3 uses too much registers in this case. I don't remember the exact error message, but one of the asm statement clobbers half of the x86 register set and CLang complains.

Reply:
Interesting. Mind adding a small comment?